### DISCUSSION TITLE
Windows Portable version

### DISCUSSION THREAD

**Discussion by luckedea** [nzbgetcom/nzbget#47] [Ideas]
Please comment if you are interested in Windows Portable version. Thank you.

**Comment by timojinn** [comment]
I always choose Portable when given the option, so it's definitely a yes from me.  It's always somewhat irritating to me when apps use the User Profile for things, instead of just keeping them in the program folder.  I dislike when I need to hunt down multiple locations.  Also, it would be nice to have the option to run it off a USB drive that could easily move from PC to PC if a problem arises, thus maintaining continuity grabbing content.

**Reply by luckedea** [reply]
Thank you @timojinn this makes sense. I'm working on extensions and docker at the moment and will be looking into portable too.

**Comment by MaelRivera** [comment]
Thanks a lot for your work. 
A Portable version with a standalone ServiceInstall and ServiceUninstall - or a batch file would be great.

**Comment by d4k0** [comment]
I would also like to see a portable version of NZBGet. The program already seems to be quite portable, as I have been extracting the files from the installer for years and have NZBGet write the settings to the nzbget.conf file in the program directory.

The paths can be adjusted using several variables so that temporary files are located in a subdirectory of NZBGet. These are explained here:

https://trash-guides.info/Downloaders/NZBGet/Basic-Setup/

The original developer wrote in 2015 that NZBGet writes settings for the tray icon to the registry, but I could not find these entries in my system:

```
HKEY_CURRENT_USER, "Software\\NZBGet":
"ShowTrayIcon"
"ShowConsole"
"ShowWebUI"
```
Source: https://forum.nzbget.net/viewtopic.php?p=13080&sid=9e7f2b7782876f72814d4ce205508677#p13080
